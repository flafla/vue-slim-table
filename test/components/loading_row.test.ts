import { mount } from '@vue/test-utils'
import LoadingRow from '@/ts/components/loading_row.vue'

test('mount component', async() => {
  const origRandom = Math.random
  Math.random = () => 1

  expect(LoadingRow).toBeTruthy()
  const wrapper = mount(LoadingRow, { props: { columnsLength: 7 } })
  expect(wrapper.html()).toMatchSnapshot()

  Math.random = origRandom
})
