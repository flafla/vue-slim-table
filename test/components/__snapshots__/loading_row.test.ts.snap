// Vitest Snapshot v1

exports[`mount component 1`] = `
"<tr>
  <td colspan=\\"7\\" class=\\"vst-loading-row vst-loading-row-11\\">
    <div></div>
  </td>
</tr>"
`;
